Makefile.include

```diff
@@ -213,7 +216,7 @@ all: ..compiler-check ..build-message $(USEPKG:%=${BINDIR}%.a) $(APPDEPS)
 	$(AD)DIRS="$(DIRS)" "$(MAKE)" -C $(CURDIR) -f $(RIOTBASE)/Makefile.application
 ifeq (,$(RIOTNOLINK))
 ifeq ($(BUILDOSXNATIVE),1)
-	$(AD)$(if $(CPPMIX),$(CXX),$(LINK)) $(UNDEF) -o $(ELFFILE) $(BASELIBS) $(LINKFLAGS) $(LINKFLAGPREFIX)-no_pie
+	$(AD)$(if $(CPPMIX),$(CXX),$(LINK)) $(UNDEF) -o $(ELFFILE) $(filter-out $(EMPTYARCHIVES), $(BASELIBS)) $(LINKFLAGS) $(LINKFLAGPREFIX)-no_pie
```

Comment:
Please try
```make
$(shell find $(BASELIBS) -size +8c)
```
which should return the .a files that are bigger than eight bytes. The .a header is 8 bytes.